### ISSUE TITLE
Move patterns from the wiki to markdown files in the repo

### ISSUE THREAD

**Issue by spier** [InnerSourceCommons/InnerSourcePatterns#132]
Currently we have patterns in various places:

* **markdown** (.md) files in the repo (true for most of the Proven/Reviewed patterns)
* **gDocs**
* **wiki** of this repo

There are some challenges with this:

a) it is harder to find the patterns (as one has to look in multiple places)
b) providing feedback for the patterns in the wiki is not possible (as the wiki does not get forked together with the rest of the repo)
c) providing feedback for the patterns in gDocs might not always be possible (depending on how the owner of the gDoc has set the permissions)

Therefore I propose to move all patterns to the repo, irrespective of the **Maturity** or **Review Step** they are in.

This change will empower all contributors to use the same contribution process for any pattern _i.e. fork + edit + pull request_.

To slice the problem a bit smaller, I suggest to start by moving the patterns **from the wiki to markdown files in the repo.**

## References

[mini discussion in Slack](https://innersourcecommons.slack.com/archives/C2EFRTS6A/p1584554578044500)

**Comment by spier** [comment]
Links to the wiki:

```
$ grep "innersourcecommons.org/wiki" README.md
* [Overcome Acquisition Based Silos - Developers](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Overcome-Acquisition-based-Silos)
* [Overcome Acquisition Based Silos - Managers](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Overcome-Acquisition-based-Silos)
* [Shared Code Repo Different from Build Repo](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Shared-Code-Repo-Different-from-Build-Repo) - *Deal with the overhead of having shared code in a separate repository that isn't the same as the project-specific one that is tied to production builds.*
* [Incentive Alignment](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Donut:-Creating-Developer-Incentive-Alignment-for-InnerSource-Contribution)
* [Change the Developers Mindset](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Pattern:-change-the-developers-mindset)
* [Share Your Code to Get More Done - Likely Contributors Variant](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Pattern:-Share-Your-Code-to-Get-More-Done---Likely-Contributors-Variant)
* [Donut 3: How to Defeat the Hierarchical Constraints](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Donut-3%3A-how-to-defeat-the-hierarchical-constraints)
* [Donut 5: Project Management Time Pressures](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Donut-5:-project-management-time-pressures)
* [Donut 6: Organizational Mindset Change](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Donut-6:-organizational-mindset-change)
* [Donut 8: Not Invented Here](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Donut-8:-Not-invented-here)
* [Donut: Bad Weather For Liftoff](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Donut:-Bad-weather-for-liftoff)
```

**Comment by rrrutledge** [comment]
Let's do it.  Sounds great!

**Comment by spier** [comment]
In https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/134 we ported a first set of wiki pages from innersourcecommons.org/wiki to InnerSourcePatterns (this repo).

In https://github.com/InnerSourceCommons/innersourcecommons.org/issues/134 those ported wiki pages should get deleted. (By chance it is issue number 134 in both repos. Don't get confused here :))

Once that is done, I suggest to go through the rest of the [innersourcecommons.org/wiki][wiki] and port all remaining patterns into the InnerSourcePatterns repo.

To summarize the next steps again:

- [x] delete the already ported wiki pages from [innersourcecommons.org/wiki][wiki]
- [ ] port all remaining patterns from [innersourcecommons.org/wiki][wiki] into markdown files in this repo 

[wiki]: https://github.com/InnerSourceCommons/innersourcecommons.org/wiki

**Comment by lenucksi** [comment]
Deleted the pages mentioned in https://github.com/InnerSourceCommons/innersourcecommons.org/issues/134

**Comment by spier** [comment]
@lenucksi I checked, and could not find any other Patterns there that can be ported.

However I found more things that can be cleaned up in the wiki [innersourcecommons.org/wiki](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki). 

Posted it here to save myself some time.

## Patterns already ported to repo

- [x] [wiki/Junkyard-Styled-Inner-Sourcing](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Junkyard-Styled-Inner-Sourcing)  can be deleted, has moved to [junkyard-styled-innersourcing.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/junkyard-styled-innersourcing.md)
- [x] [wiki/Open-Source-trumps-InnerSource](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Open-Source-trumps-InnerSource) can deleted, pending PR https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/46
- [x] [wiki/Review-Committee--aka--Cheese-Interface](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Review-Committee--aka--Cheese-Interface) can deleted, has been moved to  [review-committee.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/review-committee.md)

## Non-patterns pages that are no longer needed

- [x] [wiki/Pattern-States](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/Pattern-States) can be deleted, moved to [meta/pattern-states.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/pattern-states.md)
- [x] [wiki/InnerSource-Patterns-template](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/InnerSource-Patterns-template) can be deleted, moved to [meta/pattern-template.md](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/pattern-template.md)
- [x] [wiki/InnerSource-Patterns](https://github.com/InnerSourceCommons/innersourcecommons.org/wiki/InnerSource-Patterns) can be deleted, there is https://github.com/InnerSourceCommons/InnerSourcePatterns now :)

**Comment by spier** [comment]
if you can do those deletions, then we can close this issue here. Nothing else to port from that wiki.

**Comment by lenucksi** [comment]
> if you can do those deletions, then we can close this issue here. Nothing else to port from that wiki.

All done, closing this.
